### ISSUE TITLE
Models (ORM) page is hard to follow (structure)

### ISSUE THREAD

**Issue by Chealer** [phalcon/docs#2603]
There is still a lot of it I haven't understood, but I already had much difficulty following [the Models page](https://docs.phalcon.io/4.0/en/db-models).

I would say difficulty starts right with the first statement of the first example:
`use MyApp\Models\Invoices;`
At this point one thinks that class must be defined in an earlier documentation page. But as far as I see, that is not the case. In fact, that class is defined later in that page, in [section _Creating Models_](https://docs.phalcon.io/4.0/en/db-models#creating-models).

One natural solution would be to move section Creating Models earlier, but since the section where that reference occurs is the Overview section, that would mean the Overview section would come second, which may not make sense.

I suggest either:

1. integrating Creating Models in Overview,
2. duplicating part of Creating Models in Overview,
3. or moving Creating Models first

**Comment by niden** [comment]
@Chealer Have a look at this:

https://github.com/phalcon/docs/commit/55f3917518d43483ab2ae8bb57a0970befc09b23

and the relevant docs page:

https://docs.phalcon.io/4.0/en/db-models

**Comment by Chealer** [comment]
Thank you @niden 
My report was vague but the change to en/db-models.md does address the one issue I highlighted.

**Comment by niden** [comment]
The left menu is something we are aware and have been working on with our friends at O2. We will get something up soon

**Comment by Chealer** [comment]
Great news, thanks @niden 
Sorry, I meant to close this ticket in my last comment.
